resolwe/flow/managers/listener.py

```diff
@@ -549,6 +557,7 @@ def _make_stats(self):
         try:
             serialized = json.dumps(snapshot)
             await self._call_redis(aioredis.Redis.set, state.MANAGER_LISTENER_STATS, serialized)
+            await self._call_redis(aioredis.Redis.expire, state.MANAGER_LISTENER_STATS, 3600)
```

Comment:
Please, add a comment above this line explaining why this is needed and why here.